modules/light-clients/08-wasm/keeper/migrations.go

```diff
@@ -0,0 +1,73 @@
+package keeper
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/cosmos/ibc-go/modules/light-clients/08-wasm/internal/ibcwasm"
+	"github.com/cosmos/ibc-go/modules/light-clients/08-wasm/types"
+)
+
+// Migrator is a struct for handling in-place store migrations.
+type Migrator struct {
+	keeper Keeper
+}
+
+// NewMigrator returns a new Migrator.
+func NewMigrator(keeper Keeper) Migrator {
+	return Migrator{
+		keeper: keeper,
+	}
+}
+
+// MigrateChecksums migrates the wasm store from using a single key to
+// store a list of checksums to using a KeySet to store the checksums.
```

Comment:
just to add some more info on where `KeySet` is coming from 

```suggestion
// store a list of checksums to using a collections.KeySet to store the checksums.
```